common/clients/base_client.py

```
@@ -0,0 +1,137 @@
+
+        if response.status_code == 204:
+            return None
+
+        if response.is_success:
+            return response.json()
+
+        error_body: dict = {}
+        try:
+            error_body = response.json()
+        except Exception:
+            pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)